Connect.php

```diff
@@ -1,23 +1,23 @@
 <?php
-
-$servername = "localhost";
-$username = "root";
-$password = "";
-
-
-// $servername = "sql202.epizy.com";
-// $username = "epiz_23626301";
-// $password = "wtn2rmj3";
-// Create connection
-$con = new mysqli($servername, $username, $password,'lrr');
-
-// Check connection
-if ($con->connect_error) {
-    die("Database Connection failed: " . $conn->connect_error);
-} 
-// else
-// {
-
-// echo "Connected";
-// }
-session_start();
\ No newline at end of file
+//
+//$servername = "localhost";
+//$username = "root";
+//$password = "";
+//
+//
+//// $servername = "sql202.epizy.com";
+//// $username = "epiz_23626301";
+//// $password = "wtn2rmj3";
+//// Create connection
+//$con = new mysqli($servername, $username, $password,'lrr');
+//
+//// Check connection
+//if ($con->connect_error) {
+//    die("Database Connection failed: " . $conn->connect_error);
+//}
+//// else
+//// {
+//
+//// echo "Connected";
+//// }
+//session_start();
```

Comment:
@dzr201732120115 

Why comment out these?

If I merge these changes, then it would break the current build.


**Hui**